package.json

```diff
@@ -36,6 +36,7 @@
     "jest-pnp-resolver": "1.0.1",
     "jest-resolve": "23.6.0",
     "mini-css-extract-plugin": "0.4.3",
+    "normalize.css": "^8.0.1",
```

Comment:
You used `npm` to install this right? `npm` is a package manager, we're using `yarn`, and it's not good to mix dependencies. So I would remove `npm uninstall` the normalize package, and reinstall it with yarn.

The reason why I knew you installed with `npm` is because a new `package-lock.json` was generated. We already have a `yarn.lock` which is the equivalent of `yarn`'s version. We don't want them to conflict so you should remove `package-lock.json`.

Reply:
I did use "yarn add normalize.css". The package.json file might have been created when I, in the beginning of the quarter, mistakenly did  "npm install". 

I also checked the package-lock.json file. On my Mac, it says it was last modified on February 2! 
![image](https://user-images.githubusercontent.com/45131126/53309785-9e929a80-385e-11e9-8af9-f57a66b37c11.png)

That is weird, given that a change did show up on the commit. Anyway, I will go ahead and remove the package-lock.json file.

On a side note, I am not using the dependency anyway. I copied the content into a new .css file and appended the personalized formatting. (The normalize.css file that is imported in index.js is actually in the src folder!)